cli/lib/index.js

```diff
@@ -5,14 +5,24 @@ const prog = require('caporal');
 const encrypt = require('./actions/encrypt');
 const regexGuid = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
 
-const { ColorfulChalkLogger, DEBUG } = require('colorful-chalk-logger');
- 
+const { ColorfulChalkLogger, INFO } = require('colorful-chalk-logger');
+
+// translate to ColorfulChalkLogger log level
+if (process.argv.indexOf('--verbose')) {
+  process.argv = process.argv.filter(arg => arg != '--verbose');
```

Comment:
Hacky, but nice!

Reply:
Since we can't delete `--verbose` from caporal, I've decided to drop the `--log-levels`, otherwise it'll be very confusing to have 2 settings from log levels.